docs/src/components/home/sections/HeroSection.tsx

```diff
@@ -22,6 +22,15 @@ import { FRAMEWORKS } from '@/data/frameworks';
 import { FrameworkLogo } from '@/components/Logo';
 import { TerminalCommand } from '@/components/InstallScripts';
 
+const INSTALL_SCRIPTS = {
+  react: `npm i @aws-amplify/ui-react aws-amplify`,
+  vue: `npm i @aws-amplify/ui-vue aws-amplify`,
+  angular: `npm i @aws-amplify/ui-angular aws-amplify`,
+  flutter: 'flutter pub add amplify_authenticator',
+  // add other deps here
+  'react native': `npm i @aws-amplify/ui-react-native aws-amplify`,
```

Comment:
Will be adding more deps here that we rely on that have native dependencies (for example `react-native-safe-area-context`)

Reply:
Ln 30 is mostly a note for me 😄